.github/workflows/validate-stacks.yaml

```diff
@@ -118,20 +130,32 @@ jobs:
   odov3:
     name: with odo v3
     runs-on: ubuntu-latest
+    needs: [odov2, non-terminating, validate-devfile-schema]
```

Comment:
Not sure if we should depend on odov2 tests passing in order to run odov3 tests.

I agree with non-terminating and devfile schema validation, though if we do I think this should also be the case for odov2 tests.

Reply:
I think I can have a workaround that uses conditional statements to have `odov3` run after `odov2` even if `odov2` fails, but I need to look into it because the naive way doesn't let you cancel the workflow if it hangs. The current setup of `odov3` depending on `odov2` is because it needs all the resources it can get. I'll look into how I can skirt around it

Reply:
My latest commit that adds `if: success() || failure()` will have `odov3` run even if `odov2` fails but will make sure it always runs AFTER `odov2` so it can have the resources.

Reply:
> make sure it always runs AFTER `odov2` so it can have the resources.

Just for note keeping, this will ensure _all_ workflow jobs defined under `needs` runs first but will allow `odov3` to run regardless if any of the others fail or not.